app/javascript/src/Policies/util.jsx

```diff
@@ -56,8 +56,9 @@ function isPolicyChainChanged (chain: ChainPolicy[], originalChain: ChainPolicy[
     return true
   }
 
-  for (const policy of chain) {
-    const originalPolicy = originalChain.find(p => p.uuid === policy.uuid)
+  for (let i = 0; i < chain.length; i++) {
```

Comment:
These options are more efficient:

```suggestion
  for (let i = 0, length = chain.length; i < length; i++) {
```

```javascript
  for (let i = chain.length - 1; i >= 0 ; i--) {
```

Reply:
Good! However the length can be declared even earlier. I'll change it.

Reply:
+1 for length caching